CONTRIBUTING.md

```diff
@@ -293,20 +293,58 @@ python setup.py build_sphinx
 
 ### Flask server
 
-Make sure your machine meets the [OS dependencies](https://superset.incubator.apache.org/installation.html#os-dependencies) before following these steps.
+#### OS Dependencies
+
+Make sure your machine meets the [OS dependencies](https://superset.incubator.apache.org/installation.html#os-dependencies) before following these steps. 
+
+In addition you will need to install the mysql client to get mysql_config, which is required by mysqlclient in requirements-dev.txt.
```

Comment:
I don't think we should require mysql to be installed to do development. The solution here is probably to remove mysqlclient from `requirements-dev.txt` or use a different version that does not require have the underlying libs / mysql_config.

Reply:
Good idea. I'll roll removing it into this PR.